test/e2e/tests/add-account.spec.js

```diff
@@ -195,4 +195,64 @@ describe('Add account', function () {
       },
     );
   });
+
+  it('It should be possible to remove an account imported with a private key, but should not be possible to remove an account generated from the SRP imported in onboarding', async function () {
+    const testPrivateKey =
+      '14abe6f4aab7f9f626fe981c864d0adeb5685f289ac9270c27b8fd790b4235d6';
+
+    await withFixtures(
+      {
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        await driver.delay(regularDelayMs);
+
+        await driver.clickElement('.account-menu__icon');
+        await driver.clickElement({ text: 'Create Account', tag: 'div' });
+        await driver.fill('.new-account-create-form input', '2nd account');
+        await driver.clickElement({ text: 'Create', tag: 'button' });
+
+        await driver.clickElement(
+          '[data-testid="account-options-menu-button"]',
+        );
+
+        const menuItems = await driver.findElements('.menu-item');
+        assert.equal(menuItems.length, 3);
```

Comment:
May be add comment above that there are 3 menu items as remove option is missing.